src/components/FilterSidebar/FilterSidebar.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, DropDown, GU, textStyle, useTheme } from '@1hive/1hive-ui'
+import ListFilter from './ListFilter'
+import { useWallet } from '../../providers/Wallet'
+
+const FilterSidebar = React.memo(
+  ({
+    itemsStatus,
+    itemsSupport,
+    itemsType,
+    proposalStatusFilter,
+    proposalSupportFilter,
+    proposalTypeFilter,
+    onClearFilters,
+    onStatusFilterChange,
+    onSupportFilterChange,
+    onTypeFilterChange,
+  }) => {
+    const theme = useTheme()
+    const { account } = useWallet()
+    const supportFilterDisabled = proposalStatusFilter > 1 || !account
```

Comment:
we might want to add a comment here saying what is `1`?

Reply:
Ah yes, we should probably use the constant from filter-utils actually, good catch